src/test/java/com/exasol/adapter/document/files/AbsDocumentFilesAdapterIT.java

```diff
@@ -0,0 +1,116 @@
+package com.exasol.adapter.document.files;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.*;
+import java.nio.file.*;
+import java.sql.Statement;
+import java.util.Map;
+import java.util.concurrent.TimeoutException;
+import java.util.function.Supplier;
+
+import com.azure.core.util.BinaryData;
+import org.jetbrains.annotations.NotNull;
+import org.junit.jupiter.api.*;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import com.exasol.adapter.document.files.abstestsetup.*;
+import com.exasol.bucketfs.BucketAccessException;
+import com.exasol.dbbuilder.dialects.DatabaseObjectException;
+import com.exasol.dbbuilder.dialects.exasol.ConnectionDefinition;
+import com.exasol.dbbuilder.dialects.exasol.VirtualSchema;
+import com.exasol.exasoltestsetup.ExasolTestSetup;
+import com.exasol.exasoltestsetup.ExasolTestSetupFactory;
+import com.exasol.exasoltestsetup.testcontainers.ExasolTestcontainerTestSetup;
+
+@Tag("integration")
+@Testcontainers
+class AbsDocumentFilesAdapterIT extends AbstractDocumentFilesAdapterIT {
+    private static IntegrationTestSetup SETUP;
```

Comment:
```suggestion
    private static IntegrationTestSetup setup;
```
since it's not static final